### PR TITLE
Change pub pages to show the new dart tool

### PR DESCRIPTION
**Pull request by felpsisonfire**
@kwalrath Can you check this out, please?

### CODE REVIEW

**Review by kwalrath — COMMENTED** [review]
We aren't ready to make visible changes yet, so I'll hold onto this PR for a while.

Did you test the commands to make sure they acted like the doc says they do?

**Comment by felpsisonfire** [comment]
> We aren't ready to make visible changes yet, so I'll hold onto this PR for a while.
> 
> Did you test the commands to make sure they acted like the doc says they do?

At the end of the day, I think I still got the instructions wrong. Sorry about that, this is my first contribution ever. 
So, actually I need to test the commands and do comment the .md file with a description of the behavior and if it does match with the documentation. Right?

**Comment by kwalrath** [comment]
> So, actually I need to test the commands and do comment the .md file with a description of the behavior and if it does match with the documentation. Right?

Yes, exactly! The main goal is to test the new `dart` command. A secondary (but still important) goal is preparing to update the docs with the right commands.

I think for these files, if your testing confirms that changing `pub` to `dart pub` is all that's necessary, all you need to do is to add a comment about that to each `.md` file. Maybe something like this:

```
{% comment %}
NEW DART COMMAND:
Throughout this file, everywhere you see `pub outdated`, change it to `dart pub outdated`.
Tested in dart version 2.10.0-59.0.dev (dev)
{% endcomment %}
```

If there are any surprises, then please create an issue in this repo (for dart.dev issues) or in github.com/dart-lang/sdk (for SDK/help issues), and refer to #2617 from that issue.

File: src/tools/pub/cmd/pub-deps.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ dependencies:
   markdown: ^0.7.2
 {% endprettify %}
 
-Here's an example of the `pub deps` output for markdown_converter:
+Here's an example of the `dart pub deps` output for markdown_converter:
 
 ```terminal
-$ pub deps
+$ datt pub deps
```
Comment (sumandipanshu) [inline src/tools/pub/cmd/pub-deps.md:34]: You should correct it to `dart pub deps`

File: src/tools/pub/cmd/pub-outdated.md [hunk 2]
```diff
@@ -1,65 +1,69 @@
...
 an example that has several out-of-date dependencies.
 Three of the dependencies (`args`, `http`, and `path`) are direct,
 and one is transitive (`meta`).
 As the following example shows,
-`pub outdated` colorizes the output by default
+`dart pub outdated` colorizes the output by default
 when you run it on the command line.
 
-{% include_relative _pub-outdated-output.html %}
+{% include_relative _dart-pub-outdated-output.html %}
```
Comment (sumandipanshu) [inline src/tools/pub/cmd/pub-outdated.md:66]: "_pub-outdated-ouput.html" references to a file on the relative path. So I think you should not change it to "_dart-pub-outdated-ouput.html" as there is no file with this name on the relative path

File: src/tools/pub/cmd/pub-deps.md (hunk 1 above)
Comment (sumandipanshu) [inline src/tools/pub/cmd/pub-deps.md:34]: You should correct it to `dart pub deps`

File: src/tools/pub/cmd/pub-outdated.md [hunk 3]
```diff
@@ -161,13 +165,13 @@ it should be possible to update this package.
...
 
-The output of `pub outdated` has four columns of version information
+The output of `dart pub outdated` has four columns of version information
 for each out-of-date dependency.
 Here is the part of the [example](#example) output
 that shows the four version columns:
 Current, Upgradable, Resolvable, and Latest.
 
-{% include_relative _pub-outdated-output-columns.html %}
+{% include_relative _dart-pub-outdated-output-columns.html %}
```
Comment (sumandipanshu) [inline src/tools/pub/cmd/pub-outdated.md:174]: "_pub-outdated-ouput-columns.html" references to a file on the relative path. So I think you should not change it to "_dart-pub-outdated-ouput-columns.html" as there is no file with this name on the relative path

**Comment by kwalrath** [comment]
I'm going to close this and unassign all but one of the pages, so the conversion task is simpler and we can divvy up the rest of the pages to whoever's willing.

**Pull request closed without merging**